{
  "uid" : "35c83249f4ab0d10",
  "name" : "create Kafka datasource with form",
  "fullName" : "test_datasource.TestDataSource#test_kafka",
  "historyId" : "93cd385d45fab8e0a4ef410091f20aa6",
  "time" : {
    "start" : 1666173287348,
    "stop" : 1666173287349,
    "duration" : 1
  },
  "status" : "broken",
  "statusMessage" : "AttributeError: item not found in .env file or env",
  "statusTrace" : "self = <test.Args object at 0x7f8cb02a9490>, item = 'kafka.HOST'\n\n    def __getitem__(self, item) -> T:\n    \n        try:\n            key_1, key_2 = item.split(SEP)\n>           result = self.get_value(key_1)[key_2]\n\n__init__.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ncls = <class 'test.Args'>, k = 'kafka'\n\n    @classmethod\n    def get_value(cls, k):\n>       return cls._ini_read[k]\n\n__init__.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <configparser.ConfigParser object at 0x7f8cb02a4890>, key = 'kafka'\n\n    def __getitem__(self, key):\n        if key != self.default_section and not self.has_section(key):\n>           raise KeyError(key)\nE           KeyError: 'kafka'\n\n/usr/lib/python3.7/configparser.py:958: KeyError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test.test_datasource.TestDataSource object at 0x7f8cae40d8d0>\n\n    @allure.title(\"create Kafka datasource with form\")\n    def test_kafka(self):\n        test_ds = DataSource(\"kafka\", f\"test_kafka_{random_str()}\")\n>       test_ds.nameSrvAddr(env['kafka.HOST'])\n\ntest_datasource.py:65: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <test.Args object at 0x7f8cb02a9490>, item = 'kafka.HOST'\n\n    def __getitem__(self, item) -> T:\n    \n        try:\n            key_1, key_2 = item.split(SEP)\n            result = self.get_value(key_1)[key_2]\n        except KeyError:\n            if not os.getenv(item):\n>               raise AttributeError(\"item not found in .env file or env\")\nE               AttributeError: item not found in .env file or env\n\n__init__.py:35: AttributeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "feature",
    "value" : "datasource"
  }, {
    "name" : "suite",
    "value" : "test_datasource"
  }, {
    "name" : "subSuite",
    "value" : "TestDataSource"
  }, {
    "name" : "host",
    "value" : "1466879579e8"
  }, {
    "name" : "thread",
    "value" : "879-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_datasource"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 8,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 8
      },
      "items" : [ {
        "uid" : "af4fe0d8de277b9b",
        "status" : "broken",
        "statusDetails" : "AttributeError: item not found in .env file or env",
        "time" : {
          "start" : 1666171799586,
          "stop" : 1666171799586,
          "duration" : 0
        }
      }, {
        "uid" : "b2a578dc084468a1",
        "status" : "broken",
        "statusDetails" : "AttributeError: item not found in .env file or env",
        "time" : {
          "start" : 1666169830608,
          "stop" : 1666169830608,
          "duration" : 0
        }
      }, {
        "uid" : "e2eb54d6144cbe03",
        "status" : "broken",
        "statusDetails" : "AttributeError: item not found in .env file or env",
        "time" : {
          "start" : 1666167303988,
          "stop" : 1666167303988,
          "duration" : 0
        }
      }, {
        "uid" : "a91f99fbbd434e83",
        "status" : "broken",
        "statusDetails" : "AttributeError: item not found in .env file or env",
        "time" : {
          "start" : 1666148944901,
          "stop" : 1666148944901,
          "duration" : 0
        }
      }, {
        "uid" : "c6b840d298d05ef",
        "status" : "broken",
        "statusDetails" : "AttributeError: item not found in .env file or env",
        "time" : {
          "start" : 1666108093980,
          "stop" : 1666108093981,
          "duration" : 1
        }
      }, {
        "uid" : "b28b20ea01bc4c34",
        "status" : "broken",
        "statusDetails" : "AttributeError: item not found in .env file or env",
        "time" : {
          "start" : 1666094834037,
          "stop" : 1666094834038,
          "duration" : 1
        }
      }, {
        "uid" : "9d4173c5d42fab64",
        "status" : "broken",
        "statusDetails" : "AttributeError: item not found in .env file or env",
        "time" : {
          "start" : 1666090870991,
          "stop" : 1666090870991,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "35c83249f4ab0d10.json",
  "parameterValues" : [ ]
}